Makefile

```diff
@@ -7,5 +7,9 @@ default:
 	echo "Building armbian.."
 	$(MAKE) -C armbian
 
+dev-jekyll:
```

Comment:
Maybe check if Docker is installed?

Reply:
I've added a check that errors if `which docker` doesn't return a result.

I've also renamed the rule to `docker-jekyll` to more clearly hint that Docker will be required to run the target. I can keep the check as well if you feel it's necessary with the name change, but let me know your preference.

Reply:
That's perfect, thanks!